rclcpp_action/test/benchmark/benchmark_action_client.cpp

```diff
@@ -310,8 +310,8 @@ BENCHMARK_F(ActionClientPerformanceTest, async_cancel_goal)(benchmark::State & s
     rclcpp::spin_until_future_complete(node, future_cancel, std::chrono::seconds(1));
     auto cancel_response = future_cancel.get();
 
-    using CancelResponse = test_msgs::action::Fibonacci::Impl::CancelGoalService::Response;
-    if (CancelResponse::ERROR_NONE != cancel_response->return_code) {
+    using CancelActionResponse = test_msgs::action::Fibonacci::Impl::CancelGoalService::Response;
```

Comment:
I have no issue with this change, but why is it necessary?

Reply:
It is shadowing a variable defined here: https://github.com/ros2/rclcpp/blob/9bfb9c574c1594287424c0f6ca5c776e986b10e7/rclcpp_action/test/benchmark/benchmark_action_client.cpp#L27